### PR TITLE
Styling issues

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/ftw.simplelayout/issues/397

26127c58-a7ea-11e6-9f66-6a10957b11b4
screen shot 2016-11-11 at 08 41 24

### CODE REVIEW

**Comment by maethu** [comment]
@bierik Simplelayout should not depend on blueberry? it know depends on ftw.theming, which is OK

**Comment by bierik** [comment]
@maethu updated

**Comment by bierik** [comment]
@maethu updated

File: ftw/simplelayout/browser/resources/integration.theme.scss [hunk 1]
```diff
@@ -294,17 +294,23 @@ ul[class^="sl-toolbar"] {
...
-  padding-right: $editbar-height;
+  @include screen-small() {
+    padding-right: $editbar-height;
+  }
 }
 
 .sl-toolbox {
   @include noselect();
+  @include screen-small() {
+    display: table;
```
Comment (bierik) [inline ftw/simplelayout/browser/resources/integration.theme.scss]: @maethu according to http://caniuse.com/#feat=css-table there are no issues with IE11 concerning table layout

**Pull request merged**